ONSdigital/dp-census-atlas
# Feature/new cantabular metadata and population legend strings

### WHAT

commit db55920c2db6a1981354e53b64f31e4795283b1d (HEAD -> feature/new-cantabular-metadata-and-population-legend-strings, origin/feature/new-cantabular-metadata-and-population-legend-strings)
Author: Vivian Allen <vivian.allen@methods.co.uk>
Date:   Tue Oct 25 11:43:27 2022 +0100

    one person household -> one-person houshold in content, misc content fixes

commit 261ad3ad446d371b10591d584a1154931ca3067e
Author: Vivian Allen <vivian.allen@methods.co.uk>
Date:   Tue Oct 25 10:44:08 2022 +0100

    update legend strings for population variable group

commit b5beeb912ba9d57f69888b5e77c61ce59932d0f1 (origin/develop, develop)
Merge: ff22288 dd795af
Author: Vivian Allen <vivian.allen@methods.co.uk>
Date:   Mon Oct 24 15:57:43 2022 +0100

    Merge branch 'feature/feature/hh_multi_ethnic_group-DEM-to-EILR' into develop
    
### HOW TO REVIEW

- pull code locally
- run `npm run dev` to run census-maps with new content jsons
- check out the 'population' variable group, see if everything looks good
